File: src/components/pages/projects/$projectId/shared/connect-snippets/swift/vapor/Sources/appwrite.swift
import Appwrite
import Foundation

let env = ProcessInfo.processInfo.environment

let project = Project(
  Client()
    .setEndpoint(env["APPWRITE_ENDPOINT"]!)
    .setProject(env["APPWRITE_PROJECT_ID"]!)
    .setKey(env["APPWRITE_API_KEY"]!)
)

File: src/components/pages/projects/$projectId/shared/connect-snippets/kotlin/ktor/src/main/kotlin/Appwrite.kt
import io.appwrite.Client
import io.appwrite.services.Project

val appwrite = Project(
  Client()
    .setEndpoint(System.getenv("APPWRITE_ENDPOINT"))
    .setProject(System.getenv("APPWRITE_PROJECT_ID"))
    .setKey(System.getenv("APPWRITE_API_KEY"))
)
